pages/Frequency/Overview.md

```diff
@@ -1,38 +1,24 @@
-# Frequency DSNP Implementation (Pre-Release)
-__Version pre-1.0.0__
+# Frequency DSNP Implementation
 
-_Final release is pending release of [Frequency v1.0.0](https://github.com/LibertyDSNP/frequency)_
-
-DSNP on Frequency is built using Message Source Accounts (MSAs) for identity and delegation with Schemas defined for each Announcement Type.
-Frequency uses Schemas to define how to create and pass messages.
-
-## Links
+## Language and Framework
+Frequency is a Polkadot Parachain written in Rust, using the Substrate framework.
 
+- [Rust-lang.org](https://www.rust-lang.org/)
+- [Getting Started - Polkadot Wiki](https://wiki.polkadot.network/docs/getting-started)
+- [Parachains - Polkadot Wiki](https://wiki.polkadot.network/docs/learn-parachains)
+- [Substrate Rust Docs](https://paritytech.github.io/substrate/master/sc_service/index.html)
 - [Frequency Documentation](https://libertydsnp.github.io/frequency/)
 - [Frequency GitHub](https://github.com/LibertyDSNP/frequency)
 
-## Schemas
-
+## DSNP Schemas
 Official DSNP Schemas may be found in [GitHub](https://github.com/LibertyDSNP/schemas).
 
-## Libraries
-<!-- yaspeller ignore:start -->
-| Name | Language(s) |
-| --- | --- |
-| [TODO]() | JavaScript/TypeScript |
-<!-- yaspeller ignore:end -->
-
 <!--- Uncomment for pre-release changes and prefix the version with `pre-[next version]`
 ## Prerelease Changelog
 
 - [DIP-###](https://github.com/LibertyDSNP/spec/issues/###)
 
 --->
-## Releases
-
-| Version | Description | DSNP Compatibility | Release Date | Changelog |
-| --- | --- | --- | --- | --- |
-| `pre-1.0.0` | Pre-Release | 1.1.x | 2022-09-19 | N/A |
```

Comment:
I think we can remove detailed versioning from these pages (we will always have `git blame`), but the Frequency spec _should_ state which version of DSNP it implements.

Reply:
@shannonwells Thoughts on how to do this nicely?

Reply:
| Frequency Release | DSNP Version | Release date | 
| --- | --- | --- |
| 1.1.0 | 1.1| 2022-09-19 | 

?

Reply:
I like that.

Reply:
Updated Overview page with it. @wesbiggs does that cover your concerns?

Reply:
Yep, that works.